cloud-sql/postgres/activerecord/Gemfile

```diff
@@ -2,16 +2,17 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", "~> 5.2.3"
+gem "rails", ">= 5"
 # Use postgres as the database for Active Record
-gem "pg", "~> 0.21.0"
+gem 'pg', '~> 1.3', '>= 1.3.5'
```

Comment:
Please use double quotes for strings. (This is what's causing the lint errors which is why the Ruby Oldest CI job is failing.)

Reply:
Done.